src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift

```diff
@@ -923,6 +923,7 @@ class HomeTableViewController: UITableViewController, NavigationBarOpacityDelega
 	private func showBackgroundFetchAlertIfNeeded(completion: @escaping () -> Void = {}) {
 		if CWAHibernationProvider.shared.isHibernationState {
 			completion()
+			return
```

Comment:
As we need a return here, I would prefer a `guard.` That's the same result, but makes the code more clearly. :)